### PR TITLE
Introduce Release Template

### LINKED ISSUE #4466: [FEATURE] for lawnchair nightly update logs, dont include the dependancy updates
### Describe the feature

including dozens of dependency updates is just clutter, could you stash them away

### Why would it be useful to add?

prevent clutter

### Did the feature exist in Lawnchair v2? (Play Store version)

Yes

### Example(s)

.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by validcube**
## Description

Discourage all bot users and also make everything look nice (maybe?)

Closes #4466.

## Type of change

✅ General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

<summary>
Fields for GitHub parser
<details>

Fixes #4466

</details>
</summary>

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        - dependencies
+    - title: 🗿 Unclassified
+      labels:
+        - '*'
+      exclude:
+        labels:
+          - enhancement
+          - dependencies
+          - github_actions
+          - housekeeping
```
Comment (Goooler) [inline .github/release.yml]: If we can fold them instead of just hiding?

Comment (osm0sis) [reply .github/release.yml]: Only including the latest entry for each dependency would be ideal.

**Comment by validcube** [comment]
🔥 @Goooler @osm0sis Thanks for the review, I'm currently very busy at the moment and the PR is already a month old, so I'm closing this in case you want to PR it yourself. Thanks again for reviewing!

File: .github/release.yml (hunk 1 above)
Comment (RustoMCSpit) [reply .github/release.yml]: > If we can fold them instead of just hiding?

does it matter

File: .github/release.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        - '*'
+      exclude:
+        labels:
+          - dependencies
+          - github_actions
+          - housekeeping
+    - title: 🧹 Housekeeping
+      labels:
+        - housekeeping
+        - github_actions
```
Comment (Goooler) [inline .github/release.yml]:
```suggestion
```

File: .github/release.yml (hunk 1 above)
Comment (Goooler) [inline .github/release.yml]:
```suggestion
```

**Review by Goooler — APPROVED** [review]

File: .github/release.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+changelog:
+  exclude:
+    labels:
+      - bot
+    authors:
+      - renovate
+      - lawnchair-bot
+      - crowdin-bot
+  categories:
+    - title: 🪄 Features
```
Comment (Goooler) [inline .github/release.yml]:
```suggestion
    - title: 🪄 Features & 🐛 Bugfixes
```

**Pull request merged**